snapshot-labs/sx-monorepo
# feat: check for custom min voting power when proposing

### Summary



Closes: #615

This PR will set the `proposal_threshold` value properly for offchain space.

This will show a warning message when the user proposition power is less than `proposal threshold` (currently, no message, and it will fail on the sequencer side)

### How to test

1. Go to a offchain space which has set a minimum score, and where you have some voting power
2. When creating a proposal, it should show the warning about insufficient VP
